docs/structure-release-notes.md

```diff
@@ -0,0 +1,48 @@
+# Release notes structure guidelines
+
+The release notes filename should follow the file naming guidelines.
```

Comment:
Would it be possible to link to these guidelines here?

Reply:
Added a doc for file naming and linked it.